Makefile

```diff
@@ -1,20 +1,27 @@
-# run makefile from repo root
+# run from repository root
+
+TEST_SUFFIX = $(shell date +%s | base64 | head -c 15)
```

Comment:
probably needs to echo out the test_suffix. or it would not be easy to figure out which log belongs to which test when something fails.

Reply:
Added.